packages/dm-core/src/components/UiRecipesSideBarSelector.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useContext, useEffect, useState } from 'react'
+
+import styled from 'styled-components'
+import { CircularProgress } from '@equinor/eds-core-react'
+import { UiPluginContext } from '../context/UiPluginContext'
+import { ErrorBoundary } from '../utils/ErrorBoundary'
+import { useBlueprint } from '../hooks'
+import { IUIPlugin } from '../types'
+
+const lightGray = '#d3d3d3'
+
+const RecipeSidebarWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding-right: 5px;
+  border-right: 2px black solid;
+`
+
+const Wrapper = styled.div`
+  align-self: start;
+  width: 100%;
+  display: flex;
+`
+
+interface ISPButton {
+  active: boolean
+}
+
+const SelectPluginButton = styled.div<ISPButton>`
+  padding: 5px 10px;
+  margin: 10px;
+  text-align: center;
+  border-bottom: ${(props: any) =>
+    (props.active == true && '2px #017078FF solid') ||
+    `2px ${lightGray} solid`};
+  &:hover {
+    background: ${lightGray};
+    cursor: pointer;
+  }
+`
+
+type TSelectablePlugins = {
+  name: string
+  component: (props: IUIPlugin) => JSX.Element
+  config: any
+}
+
+export function UiRecipesSideBarSelector(props: IUIPlugin): JSX.Element {
+  const { idReference, categories, onSubmit, onOpen, type } = props
+  const { loading, getUiPlugin } = useContext(UiPluginContext)
+  const { uiRecipes, isLoading } = useBlueprint(type)
```

Comment:
suggestion: 
rename loading to isContextLoading
and isLoading to isBlueprintLoading